apps/demo-react/components/contractTesting/Withdrawals.tsx

```diff
@@ -0,0 +1,192 @@
+    if (data) {
+      const sortedSelectedRequests = data.sortedClaimableRequests.filter(
+        (req) => selectedRequests.includes(req.stringId),
+      );
+      await claimMutation(sortedSelectedRequests);
+    }
+  };
+
+  const submit = useCallback(
+    async (_: string, resetForm: () => void): Promise<void> => {
+      await request(requests, resetForm);
+    },
+    [request, requests],
+  );
+
+  return (
+    <BlueWrapper>
+      <H3>Withdrawals</H3>
+      <DataTable>
+        <DataTableRow title="My requests" loading={!data}>
+          <RequestCounterStyled>
+            <Tooltip title="Ready to claim">
+              <span>
+                <TickSquare />
+                <span>{data?.readyCount}</span>
+              </span>
+            </Tooltip>
+          </RequestCounterStyled>
+          <RequestCounterStyled>
+            <Tooltip title="Pending">
+              <span>
+                <TimeSquare />
+                <span>{data?.pendingCount}</span>
+              </span>
+            </Tooltip>
+          </RequestCounterStyled>
+        </DataTableRow>
+      </DataTable>
+      <Select
+        value={selectedToken}
+        onChange={(value: OptionValue) => setSelectedToken(value as TOKENS)}
+      >
+        <Option value={TOKENS.STETH}>stETH</Option>
+        <Option value={TOKENS.WSTETH}>wstETH</Option>
+      </Select>
+      <Input
+        type="number"
+        onChange={handleInputChange}
+        value={inputValue}
+        step="0.00001"
+        max={
+          selectedToken === TOKENS.WSTETH
+            ? wstethBalance.data?.toString()
+            : stethBalance.data?.toString()
+        }
+        error={
+          selectedToken === TOKENS.WSTETH && +inputValue < 1
+            ? 'Min value is 1 wstETH'
+            : null
+        }
+      />
+      <Button onClick={void submit} loading={isTxPending}>
+        Request Withdrawal
+      </Button>
+      <Button onClick={claim} color="warning" disabled={!selectedRequests[0]}>
+        Claim
+      </Button>
+      <DataTable>
+        {data?.claimableAmountOfETH && (
+          <DataTableRow title="Claimable amount">
+            {`${formatBalance(data.claimableAmountOfETH, 5)} ETH`}
+          </DataTableRow>
+        )}
+        {data?.sortedClaimableRequests && (
+          <div>
+            {data?.sortedClaimableRequests.map((req) => (
+              <DataTableRow
+                loading={loading}
+                title={`ID ${req.stringId}`}
+                key={req.stringId}
+              >
+                <RequestStyled>
+                  <LinkStyled href={getNFTUrl(req.stringId, chainId)}>
+                    <External />
+                  </LinkStyled>
+                  <Checkbox
+                    name={req.stringId}
+                    label={`${formatBalance(req.amountOfStETH, 5)} ETH`}
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                    // @ts-ignore
```

Comment:
We use
`// @ts-expect-error <describe the error here>`
Also, can be fixed like this:
`checked={!!selectedRequests[Number(req.stringId)]}`